clientd3d/ownerdrw.c

```diff
@@ -341,16 +380,39 @@ void DrawOwnerListItem(const DRAWITEMSTRUCT *lpdis, Bool selected, Bool combo)
    SetBkMode(lpdis->hDC, OPAQUE);
    obj = (object_node*)lpdis->itemData;
 
-   hColorBg = GetBrush(GetItemListColor(lpdis->hwndItem, (selected? SEL_BGD : UNSEL_BGD)));
+   hColorBg = GetBrush(
+                  GetItemListColor(
+                     lpdis->hwndItem,
+                     (selected? SEL_BGD : UNSEL_BGD),
+                     item_rarity_value));
```

Comment:
It looks like item_rarity_value can only be ITEM_RARITY_GRADE_NORMAL here, so use that constant instead to make it easier for the reader.

Reply:
Changed parameter to `ITEM_RARITY_GRADE_NORMAL` constant as requested.